djangocms_bootstrap4/contrib/bootstrap4_grid/cms_plugins.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+
+from cms.models import CMSPlugin
+from cms.plugin_base import CMSPluginBase
+from cms.plugin_pool import plugin_pool
+
+from ...helpers import concat_classes
```

Comment:
because this is a package on its own (`bootstrap4_grid`), I suggest to use `djangocms_bootstrap4` directly instead of the relative import.